.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
 <!-- This is a pull request template, you do not need to uncomment or remove the comments, they won't show up in the PR text. -->
 
 <!-- Your Pull Request name should start with one of the following tags
-  [NEW] For new features
-  [IMPROVE] For an improvement (performance or little improvements) in existing features
-  [FIX] For bug fixes that affect the end-user
-  [BREAK] For pull requests including breaking changes
-  Chore: For small tasks
-  Doc: For documentation
+  feat: Adding a new feature
+  refactor: For an improvement (performance or little improvements) in existing features
+  fix: For bug fixes that affect the end-user
+  chore: For small tasks
+  doc: For documentation
```

Comment:
the correct is `docs` .. not sure if we should change to `doc` though.. 🤔 
```suggestion
  docs: For documentation
```

Reply:
Conventional commits mentions `docs` 🤷